test/parallel/test-child-process-promisified.js

```diff
@@ -32,3 +32,22 @@ const execFile = promisify(child_process.execFile);
     assert(err.message.includes('doesntexist'));
   }));
 }
+const failingCodeWithStdoutErr =
+  'console.log("out");console.error("err");process.exit(1)';
```

Comment:
I prefer not to - this string goes into the command line, and I have no idea what this will do there, especially not in windows.